nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestAttributesToJSON.java

```diff
@@ -560,8 +555,34 @@ public void testAttributeWithNestedJsonOutputAsStringInAttribute(String nestedJs
 
         String json = testRunner.getFlowFilesForRelationship(AttributesToJSON.REL_SUCCESS)
                 .get(0).getAttribute(AttributesToJSON.JSON_ATTRIBUTE_NAME);
-        ObjectMapper mapper = new ObjectMapper();
-        Map<String, Object> val = mapper.readValue(json, Map.class);
+        Map<String, Object> val = MAPPER.readValue(json, Map.class);
         assertInstanceOf(String.class, val.get(TEST_ATTRIBUTE_KEY));
     }
+
+    @Test
+    public void testAttributeWithJsonBetweenLeadingAndTrailingSpaces() throws JsonProcessingException {
+        final String jsonBetweenLeadingAndTrailingSpaces = "\n{\n" +
+                "    \"fruit\": \"Apple\",\n" +
+                "    \"size\": \"Large\",\n" +
+                "    \"color\": \"Red\"\n" +
+                "}\n";
+        assertDoesNotThrow((ThrowingSupplier<Object>) () -> MAPPER.readValue(jsonBetweenLeadingAndTrailingSpaces, Map.class));
+
+        final TestRunner testRunner = TestRunners.newTestRunner(new AttributesToJSON());
+        testRunner.setProperty(AttributesToJSON.DESTINATION, AttributesToJSON.DESTINATION_CONTENT);
+        testRunner.setProperty(AttributesToJSON.JSON_HANDLING_STRATEGY,
+                AttributesToJSON.JsonHandlingStrategy.NESTED.getValue());
```

Comment:
Since #8102 one no longer needs to pass the value of a `DescribedValue` to `TestRunner.setProperty` but can pass the `DescribedValue` directly instead.

```suggestion
                AttributesToJSON.JsonHandlingStrategy.NESTED);
```

I don't think this needs to be changed, just wanted to let you know.